### PR TITLE
social / peers styles cleanup

### PR DESCRIPTION
**Pull request by ChrisEddy**

### CODE REVIEW

**Comment by avaer** [comment]
These borders/lines are still in the menu:

Screen Shot 2020-10-15 at 2 20 29 PM

**Comment by avaer** [comment]
The selected item glitches/jumps instead of just being selected, due to some border? Additionally there are borders on the details pane buttons.

lol

**Comment by avaer** [comment]
Ending items are not aligned flush to the items grid.

Screen Shot 2020-10-15 at 2 26 37 PM

File: components/App.js [hunk 1]
```diff
@@ -229,6 +228,16 @@ export const onclickBindings = {
       }
     })
     setState({ menu });
+  },
+  'twoD-browse-card': e => {
+    const { menu } = getState();
+    menu.browse.selectedId = parseInt(e.name, 10);
+    menu.browse.items.forEach((item) => {
```
Comment (avaer) [inline components/App.js:235]: I don't think `forEach` is the right style for this.

Probably the intent is `menu.browse.selectedItem = menu.browse.items.find(...)`.

File: components/styles/trade.css [hunk 2]
```diff
@@ -1,9 +1,12 @@
 .twoD-trade {
-  overflow: auto;
-  max-height: 100%;
   color: white;
 }
 
+.twoD-trade-content {
+  overflow-y: scroll;
+  max-height: calc(90vh - 126px);
```
Comment (avaer) [inline components/styles/trade.css:7]: These magic numbers suggest there is a layout design problem in some padding.

**Review by avaer — COMMENTED** [review]
A few code comments.

**Comment by ChrisEddy** [comment]
addressed everything + more except the click card border "popping" imo its good UX its just a little gesture.

**Comment by avaer** [comment]
It's currently hard to test this since there is still an `undefined` when loading 2d inventory while having no items.

**Comment by avaer** [comment]
The icons are still popping/being resized when being clicked. The metrics should not change.

Also, should the browse page have spawn/wear buttons? I currently don't see any buttons in the Browse page details (on an empty fresh account).

**Comment by avaer** [comment]
One more thing as of this commit: the items still pop/realign in size when clicked, per https://github.com/webaverse/app/pull/146#issuecomment-709509329

**Review by avaer — APPROVED** [review]

**Pull request merged**